docs/user-guide/ze-usage.md

```diff
@@ -353,15 +353,26 @@ You can use the following functionalities when interacting with jobs:
 
 <img src="../images/ze/ZE-jobs-download-spool.gif" alt="Download Spool">
 
-### MVS/TSO Commands
+### MVS and TSO Commands
 
 #### Issue MVS commands
 
-Zowe Explorer also enables you to issue MVS commands. You can issue such commands as Allocate or Exec against a profile.
+1. Right-click on your profile and select the **Issue MVS Command** option. Select the **Zowe:Issue MVS Command** option.
 
-1. Press the **F1** key on your keyboard.
-2. Select the **Zowe:Issue MVS Command** option.
-3. Select your profile.
-4. Issue an MVS command.
+   Alternatively, press the **F1** key on your keyboard, then select "the **Zowe:Issue MVS Command** option.
+
+2. Select your profile.
```

Comment:
No need to select a profile. It's either select a saved command or enter a new command